src/main/java/seedu/address/model/eatery/Eatery.java

```diff
@@ -19,27 +17,32 @@
     private final Name name;
     // Data fields
     private final Address address;
-    private final Set<Tag> tags = new HashSet<>();
+    private final Category category;
+    private Set<Tag> tags = new HashSet<>();
```

Comment:
why remove the final keyword?

Reply:
bc the tags in the eateries will keep changing? so i don't think i should put final